pol-core/pol/create.cpp

```diff
@@ -785,7 +771,7 @@ void ClientCreateCharKR( Network::Client* client, PKTIN_8D* msg )
       chr->equip( tmpitem );
     else
     {
-      ERROR_PRINT.Format( "Create Character: Failed to equip hair 0x{:X}\n" ) << tmpitem->graphic;
+      ERROR_PRINTLN( "Create Character: Failed to equip hair {:#X}", tmpitem->graphic );
```

Comment:
Is there a difference between new fmt lib `{:#X}` and `{:#x}`? I've seen the lowercase version in the PR up until here

Reply:
As mentioned in discord X is all uppercase x is all lowercase. I would suggest because it's already inconsistent that we decide which version we want and once every logger is adapted I perform a global replace.